VagrantProvision.sh

```diff
@@ -244,21 +334,22 @@ PATH=$HOOT_HOME/bin:$PATH
 EOT
 fi
 
-if ! grep -i --quiet 'umask 002' /etc/default/tomcat6; then
-echo "Changing Tomcat umask to group write..."
+# Was 002, changing it to 000 to get rid of errors
+if ! grep -i --quiet 'umask 000' /etc/default/tomcat6; then
+echo "### Changing Tomcat umask to group write..."
 sudo bash -c "cat >> /etc/default/tomcat6" <<EOT
 # Set tomcat6 umask to group write because all files in shared folder are owned by vagrant
-umask 002
+umask 000
```

Comment:
Fixed - I think. This seems to work.
* Change umask back to 002
* Add tomcat to the vagrant group: sudo usermod -a -G vagrant tomcat6